src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -35,17 +36,18 @@ def parse_workflows_file(file_path)
       target_repository_full_name = @scm_webhook.payload.values_at(:target_repository_full_name, :path_with_namespace).compact.first
       scm_organization_name, scm_repository_name = target_repository_full_name.split('/')
 
+      # The PR number is only present in webhook events for pull requests, so we have a default value in case someone doesn't use
+      # this correctly. Here, we cannot inform users about this since we're processing the whole workflows file
+      pr_number = @scm_webhook.payload.fetch(:pr_number, 'NO_PR_NUMBER')
+
+      commit_sha = @scm_webhook.payload.fetch(:commit_sha)
+
       workflows_file_content = File.read(file_path)
       track_placeholder_variables(workflows_file_content)
 
       # Mapping the placeholder variables to their values from the webhook event payload
-      format(workflows_file_content,
-             SCM_ORGANIZATION_NAME: scm_organization_name,
-             SCM_REPOSITORY_NAME: scm_repository_name,
-             # If someone uses this placeholder variable in a workflow which runs for pull request webhook events, we have a default
-             # value even though this is wrong. Here, we cannot inform users about this since we're processing the whole workflows file
-             SCM_PR_NUMBER: @scm_webhook.payload.fetch(:pr_number, 'NO_PR_NUMBER'),
-             SCM_COMMIT_SHA: @scm_webhook.payload.fetch(:commit_sha))
+      placeholder_variables = SUPPORTED_PLACEHOLDER_VARIABLES.zip([scm_organization_name, scm_repository_name, pr_number, commit_sha]).to_h
+      format(workflows_file_content, placeholder_variables)
```

Comment:
This is covered with [specs](https://github.com/openSUSE/open-build-service/blob/2383cb8e294015ae0824ec812e9c7c046addff9b/src/api/spec/services/workflows/yaml_to_workflows_service_spec.rb#L48-L60), so it's still working like it did before.